snap/hooks/post-refresh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+set -e
+
+display_layout=$(snapctl get display-layout)
```

Comment:
Does all this code have to be duplicated between install and post-refresh? Wouldn't it make sense to have a helper script run by each?

Reply:
Yes, they did end up _almost_ the same. I'm not convinced de-duplication is enough motivation for adding a script for the common bits.